packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -87,12 +84,13 @@ const WRAPPER_PATH = 'webxdc-wrapper.45870014933640136498.html'
 
 const BOUNDS_UI_CONFIG_PREFIX = 'ui.desktop.webxdcBounds'
 
-const DEFAULT_BOUNDS_WEBXDC = {
+type Size = { width: number; height: number }
+
+const DEFAULT_SIZE_WEBXDC: Size = {
   width: 375,
   height: 667,
 }
-
-const DEFAULT_BOUNDS_MAP = {
+const DEFAULT_SIZE_MAP: Size = {
```

Comment:
Maybe there should be a space here idk, I kind of thought they belong together so I don't mind.